articles/becoming-a-flutter-developer/index.md

```diff
@@ -0,0 +1,176 @@
+
+
+### Table of Content
+- [Who is a Flutter Developer](#who-is-a-flutter-developer)
+- [Roles and Responsibilites of a Flutter Developer](#roles-and-responsibilites-of-a-flutter-developer)
+- [Flutter Developer Tools](#flutter-developer-tools)
+- [Flutter Developer Skills](#flutter-developer-skills)
+- [Mistakes to Avoid When Learning Flutter Development](#mistakes-to-avoid-when-learning-flutter-development)
+- [Getting a job as a Flutter Developer](#getting-a-job-as-a-flutter-developer)
+   - [Job Description Example](#job-description-example)
+   - [Flutter Developer Interview Questions](#flutter-developer-interview-questions)
+- [Further Reading](#further-reading)
+- [Conclusion ](#conclusion)
+
+
+### Who is a Flutter Developer
+
+A Flutter developer is a software developer who uses the Flutter framework to develop cross-platform applications. 
+
+### Roles and Responsibilities of a Flutter Developer
+
+1. **To Code Applications.**
+Flutter developers use the Flutter SDK to craft beautiful and interactive applications.
+
+2. **TO Test Applications.**
+Flutter developers test the functionality of the applications to make sure they do not crash. Testing ensures that the apps adhere to system and user requirements.
+
+3. **To Deliver Applications.**
+Flutter developers are involved in the deployment of applications. A mobile app is placed on `PlayStore` for Android applications or `AppleStore` for iOS applications. User feedback is then monitored and used to make changes to the applications.
+
+
+### Flutter Developer Tools
+
+1. **Version Control Tools**
+
+These tools allow software developers to keep track of changes they make on their projects. They also boost collaboration among software developers. As a result, they help increase overall productivity. 
+
+Examples of version control tools:
+
+[Git and Github](https://github.com/) - To use git and github you will need to install git to the code editor or IDE you are using and have an existing github account. All you need to do is write git commands to push your code to a github repository. From the repository other developers can help you improve your code. You can also set your repository as private.
+
+- [Bitbucket](https://bitbucket.org/)
+
+2. **Flutter Framework**
+
+Flutter is a UI framework used to develop high performance and high-quality native applications for Android, iOS, web and desktop using a single code base. Dart is the primary programming language for Flutter Framework. 
+
+Some features of Flutter that make it popular include 
+
+1. `Hot Reloading` which enables developers to load code changes quickley and preserves the state of the app. 
+2. `Hot Restart`. This is similar to hot reloading but causes the app to lose its state and takes more time.
+3. `Present UI element` to create beautiful app designs for better `UI and UX`. 
```

Comment:
Rephrase
``Present UI element` to create beautiful app designs for better `UI and UX`. `
To
`Availability of widgets which can be easily be imported into Flutter projects`